glide.yaml

```diff
@@ -74,4 +74,6 @@ testImport:
   version: ~7.0.0
 - package: k8s.io/api
   version: 73d903622b7391f3312dcbac6483fed484e185f8
-- package: github.com/influxdata/influxdb/client/v2
\ No newline at end of file
+- package: github.com/influxdata/influxdb/client/v2
+- package: github.com/rjtsdl/conform
```

Comment:
This is the package I am going to use to conform,
Checkout github.com/rjtsdl/conform readme, the diff of this fork compared with origin is, this fork support `redact` tag. I also sent a PR to origin, we can switch back later though.